lib/widgets/theme.dart

```diff
@@ -134,44 +97,37 @@ class DesignVariables extends ThemeExtension<DesignVariables> {
       unreadCountBadgeTextForChannel: Colors.black.withValues(alpha: 0.9),
     );
 
-  DesignVariables.dark() :
+    DesignVariables.dark() :
     this._(
-      background: const Color(0xff000000),
-      bgCounterUnread: const Color(0xff666699).withValues(alpha: 0.37),
-      bgTopBar: const Color(0xff242424),
-      borderBar: Colors.black.withValues(alpha: 0.41),
-      icon: const Color(0xff7070c2),
-      labelCounterUnread: const Color(0xffffffff).withValues(alpha: 0.7),
-      labelEdited: const HSLColor.fromAHSL(0.35, 0, 0, 1).toColor(),
-      labelMenuButton: const Color(0xffffffff).withValues(alpha: 0.85),
-      mainBackground: const Color(0xff1d1d1d),
-      title: const Color(0xffffffff),
+      background: const Color(0xff000000),  // Keeps deep black background for full dark contrast.
+      bgCounterUnread: const Color(0xff333366).withValues(alpha: 0.37),  // Darkened for better contrast with lighter text.
+      bgTopBar: const Color(0xff1a1a1a),  // Darkened for better distinction from the background.
+      borderBar: Colors.black.withValues(alpha: 0.41),  // Lower contrast for subtle visibility.
```

Comment:
It looks like this PR is trying to make an original redesign of all our dark-theme colors. That isn't what this issue calls for.

See again what Chris said at https://github.com/zulip/zulip-flutter/pull/991#issuecomment-2400660762 , the first comment above:

> 4\. In the commit(s) that change the colors that are used, the commit message should link to the source of the new colors. Greg pointed to an updated Figma in [Tweak dark theme for slightly higher contrast #973](https://github.com/zulip/zulip-flutter/issues/973); I believe that means you don't have to choose any colors yourself.